src/components/Popover/Popover.tsx

```diff
@@ -182,25 +181,28 @@ export const Popover = ({
 
   return (
     <FocusLock disabled={!activeFocus}>
-      <article
-        aria-describedby={ariaDescribedBy}
+      <div
         aria-hidden={!isActive}
-        aria-labelledby={ariaLabelledBy}
         className={componentClassName}
         onKeyDown={(e) => handleOnKeyDown(e)}
-        ref={ref}
-        role="dialog"
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+        role="button"
         tabIndex={0}
         {...other}
       >
-        <div className={styles['popover__content']}>
-          {header}
-          {body}
-          {footer}
-        </div>
-        <div className={styles['popover__arrow']}></div>
-      </article>
+        <article
+          aria-describedby={ariaDescribedBy}
+          aria-labelledby={ariaLabelledBy}
+          ref={ref}
+          role="dialog"
+        >
+          <div className={styles['popover__content']}>
+            {header}
+            {body}
+            {footer}
+          </div>
+          <div className={styles['popover__arrow']}></div>
+        </article>
+      </div>
     </FocusLock>
```

Comment:
error: Non-interactive elements should not be assigned mouse or keyboard event listeners.

Since <article> is Non-interactive element, added <div> as parent element also added event listener to parent element. 
Structure is similar as `drawer.tsx` line 212 - 234